test/fusionHost.hpp

```diff
@@ -280,14 +280,14 @@ void batchNormSpatialHostFwdTrain(const tensor<T>& input,
 }
 
 template <class T, class U>
-void batchNormSpatialHostBwdTrain(const tensor<T>& x_input,
+void batchNormSpatialHostBwdTrain(const tensor<U>& x_input,
```

Comment:
I know it's an old code and you've just rearranged the types. But may I ask to rename the types as well? It's not so clear what does `U` and `T` mean. And why some types were replaced.